cob_bringup/tools/teleop_v2.xml

```diff
@@ -10,7 +12,9 @@
 	</include>
 
 	<node pkg="cob_teleop" type="cob_teleop_v2" name="$(anon teleop_node)" output="screen" >
-		<remap from="base_controller_command" to="base/twist_controller/command"/>
+		<remap from="/base/driver/init" to="$(arg service_ns)/init"/>
+		<remap from="/base/driver/recover" to="$(arg service_ns)/recover"/>
```

Comment:
should we not simply 
```<remap from="init" to="$(arg service_ns)/init"/>```
maybe we need to change the implementation of the teleop nodes...

I guess all topics used in teleop nodes should be relative..i.e. only ```init``` or ```recover````or ```cmd_vel```